### PR TITLE
Fixed #3150 (winding pack current density constraint)

### LINKED ISSUE #3150: Incorrect error in winding pack current density constraint.
In constraint 35 for the current density in the winding pack the constraint error is calculated as
```
tmp_err = jwptf - jwptf
```
This is wrong and surely causes the constraint to never be violated? I believe it should be 
```
tmp_err =  jwptf - jwdgpro 
```

### PR DESCRIPTION
**Pull request by mkovari**
Fixed #3150.  
Removed obsolete references to PROCESS Users's Guide.

### CODE REVIEW

File: process/current_drive.py [hunk 1]
```diff
@@ -1279,8 +1278,7 @@ def iternb(self):
         fshine  : output real : shine-through fraction of beam
         This routine calculates the current drive parameters for a
         neutral beam system, based on the 1990 ITER model.
-        AEA FUS 251: A User's Guide to the PROCESS Systems Code
-        ITER Physics Design Guidelines: 1989 [IPDG89], N. A. Uckan et al,
+                ITER Physics Design Guidelines: 1989 [IPDG89], N. A. Uckan et al,
```
Comment (jonmaddock) [inline process/current_drive.py:1281]: Fix indentation please.

File: process/evaluators.py [hunk 2]
```diff
@@ -37,8 +37,7 @@ def fcnvmc1(self, n, m, xv, ifail):
         n-dimensional point of interest xv.
         Note that the equality constraints must precede the inequality
         constraints in conf.
-        AEA FUS 251: A User's Guide to the PROCESS Systems Code
-        :param n: number of variables
+                :param n: number of variables
```
Comment (jonmaddock) [inline process/evaluators.py:40]: Again, fix indentation please.

**Review by jonmaddock — CHANGES_REQUESTED** [review]
I think there's some indentation that needs correcting. Also, why has an image been put in the examples directory? Perhaps this belongs in the docs?

**Comment by mkovari** [comment]
The image files plot_proc_1, 2 and 3 must have been put into the examples folder automatically when I ran `pytest`.  Because there were so many modified files I used `git commit -a`, and because I didn't have the examples folder open in VSCode I didn't see they had been changed.  

However, they seem to fit, and since only the new third page is shown in the changes tab, I suppose that the other two pages, plot_proc_1.png and plot_proc_2.png were already there, so I have left it there.

**Comment by mkovari** [comment]
I have removed extra indentations from six files.

**Review by jonmaddock — CHANGES_REQUESTED** [review]
Please fixup the changes to comments to eliminate the 2 correcting commits and eliminate churn. The `plot_procX.png` figures should be created by running the `examples.ipynb` notebook, and the resulting pngs displayed in the notebook. Please look at how figure 1 and 2 are being handled by the notebook to match. Thanks!

**Comment by mkovari** [comment]
The plot_proc figures have nothing to do with this pull request.  They result from Chris's edit.  If the example notebook needs changing it should be done there.  I will delete the unwanted png.

**Comment by mkovari** [comment]
Unwanted png removed.  Rebased.

**Comment by mkovari** [comment]
I have fixed the conflict localy, but when I rebased and then tried to force push, I couldn't do it:
```
(env) mkovari@J1714:~/PROCESS$ git push --force
ssh: connect to host github.com port 22: Connection timed out
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

File: source/fortran/init_module.f90 [hunk 3]
```diff
@@ -111,7 +111,7 @@ subroutine init
       !! the input file, and checks the run parameters for consistency.
 
 
-      !! AEA FUS 251: A User's Guide to the PROCESS Systems Codefile:///home/mkumar/process/source/fortran/divertor_ode.f90
+      !! file:///home/mkumar/process/source/fortran/divertor_ode.f90
```
Comment (timothy-nunn) [inline source/fortran/init_module.f90:114]: Can we just remove this line, that file is long gone

**Review by timothy-nunn — DISMISSED** [review]

**Review by timothy-nunn — APPROVED** [review]

**Pull request merged**